Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -3761,6 +3761,11 @@ bool RenderLayerCompositor::isSeparated(const RenderObject& renderer)
 // into the hierarchy between this layer and its children in the z-order hierarchy.
 bool RenderLayerCompositor::clipsCompositingDescendants(const RenderLayer& layer)
 {
+    // View transition new always has composited descendants in the graphics layer
+    // tree due to hosting (but not the RenderLayer tree).
```

Comment:
```suggestion
    // View transition new always has composited descendants in the graphics layer
    // tree due to hosting (but not in the RenderLayer tree).
```

I assume this is what you meant